src/wp-includes/rest-api/endpoints/class-wp-rest-font-faces-controller.php

```diff
@@ -0,0 +1,941 @@
+		unset( $query_params['search'] );
+		unset( $query_params['search_columns'] );
+		unset( $query_params['slug'] );
+		unset( $query_params['status'] );
+
+		$query_params['orderby']['default'] = 'id';
+		$query_params['orderby']['enum']    = array( 'id', 'include' );
+
+		/**
+		 * Filters collection parameters for the font face controller.
+		 *
+		 * @since 6.5.0
+		 *
+		 * @param array $query_params JSON Schema-formatted collection parameters.
+		 */
+		return apply_filters( 'rest_wp_font_face_collection_params', $query_params );
+	}
+
+	/**
+	 * Get the params used when creating a new font face.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Font face create arguments.
+	 */
+	public function get_create_params() {
+		$properties = $this->get_item_schema()['properties'];
+		return array(
+			'theme_json_version' => $properties['theme_json_version'],
+			// When creating, font_face_settings is stringified JSON, to work with multipart/form-data used
+			// when uploading font files.
+			'font_face_settings' => array(
+				'description'       => __( 'font-face declaration in theme.json format, encoded as a string.' ),
+				'type'              => 'string',
+				'required'          => true,
+				'validate_callback' => array( $this, 'validate_create_font_face_settings' ),
+				'sanitize_callback' => array( $this, 'sanitize_font_face_settings' ),
+			),
+		);
+	}
+
+	/**
+	 * Get the parent font family, if the ID is valid.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param int $font_family_id Supplied ID.
+	 * @return WP_Post|WP_Error Post object if ID is valid, WP_Error otherwise.
+	 */
+	protected function get_parent_font_family_post( $font_family_id ) {
+		$error = new WP_Error(
+			'rest_post_invalid_parent',
+			__( 'Invalid post parent ID.', 'default' ),
+			array( 'status' => 404 )
+		);
+
+		if ( (int) $font_family_id <= 0 ) {
+			return $error;
+		}
+
+		$font_family_post = get_post( (int) $font_family_id );
+
+		if ( empty( $font_family_post ) || empty( $font_family_post->ID )
+		|| 'wp_font_family' !== $font_family_post->post_type
+		) {
+			return $error;
+		}
+
+		return $font_family_post;
+	}
+
+	/**
+	 * Prepares links for the request.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_Post $post Post object.
+	 * @return array Links for the given post.
+	 */
+	protected function prepare_links( $post ) {
+		// Entity meta.
+		return array(
+			'self'       => array(
+				'href' => rest_url( $this->namespace . '/font-families/' . $post->post_parent . '/font-faces/' . $post->ID ),
+			),
+			'collection' => array(
+				'href' => rest_url( $this->namespace . '/font-families/' . $post->post_parent . '/font-faces' ),
+			),
+			'parent'     => array(
+				'href' => rest_url( $this->namespace . '/font-families/' . $post->post_parent ),
+			),
+		);
+	}
+
+	/**
+	 * Prepares a single font face post for creation.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @return stdClass|WP_Error Post object or WP_Error.
```

Comment:
This actually never returns an error.

```suggestion
	 * @return stdClass Post object.
```

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/6034/commits/a0836f0f6b4fffc3259a2711f2ea43b02823a6e2